CONTRIBUTING.md

```diff
@@ -22,6 +23,23 @@ Retrospectives is an Azure DevOps extension. Visit [this link](https://docs.micr
 3. Publish the newly created branch to the Retrospectives repo. Use this branch as your working branch.
 4. Once you are ready to check-in, create a pull request against the 'master' branch. Link the Bug/Task that you are fixing/adding to the pull request. Reviewers will be added automatically.
 
+## Considerations
```

Comment:
@mindlessroman Adding the az cli & zip install to the Dockerfile is a good addition but I think the CONTRIBUTING.md changes may need to be reworked. This new section may give the impression that standing up a backend is required to contribute which is not the case. I like the list of dependencies (when not using the dev container) but would move to the "Backend" section. The "Backend" section may need some additional clarity on the fact that it's an optional requirement and is only needed for realtime updates. We can discuss more at stanup.

Reply:
+1 to language around env_setup.sh. Good to differentiate backend set up from frontend by adding sections for them.

Reply:
Adding some context now